src/main/java/ngelements/NGHtmlElement.java

```diff
@@ -30,4 +33,22 @@ public void waitUntilIsNotClickable(Integer secondsForTimeout) {
         WebDriverWait webDriverWait = new WebDriverWait(Context.driverManager.getDriver(), secondsForTimeout);
         webDriverWait.until(ExpectedConditions.not(ExpectedConditions.elementToBeClickable(getWrappedElement())));
     }
+
+    public void moveToElementAndClick() {
+        Actions actions = new Actions(Context.driverManager.getDriver());
+        actions.moveToElement(getWrappedElement()).click().perform();
+        log.info("Element: " + getName() + " was clicked");
+    }
+
+    public void scrollTo() {
+        js.executeScript("arguments[0].scrollIntoView();", this.getWrappedElement());
+        log.info(("Element: " + getName() + " was scrolled into view"));
+    }
+
+    public void setAttribute(String attributeName, String attributeValue) {
+        js.executeScript("arguments[0].setAttribute(arguments[1], arguments[2]);",
```

Comment:
that's why I'd like to separate all js scripts to separated class - currently you have 2 js executor fields in different classes

Reply:
I agree JSExecutor class is a reasonable solution here.

Reply:
I am using a separate class now, but I named my variable `js` - same as it was before. That's the reason why this comment is not marked as resolved